src/Graph.hpp

```
@@ -0,0 +1,25 @@
+#ifndef GRAPH_HPP
+#define GRAPH_HPP
+
+#include <string>
+#include <vector>
+
+namespace booty {
+
+class Graph {
+public:
+    Graph();
+    ~Graph();
+
+    void add_node(const std::string& label);
```

Add comments describing the failure behavior of these commands. 
For example:
What happens if the same label is passed twice?
Will it throw an error or silently fail?
How does the system handle invalid inputs?"